### ISSUE TITLE
Allow to override retries option on resilient#send and rest of methods.

### ISSUE THREAD

**Issue by ismriv** [resilient-http/resilient.js#135]
I've come across certain use cases where I would like to override the number of retries, configured in the client, on very specific requests, like in the following example, without having to create a new client instance:

    client.send(path, {retries: 0, ...}, callback);

If this is currently possible, I didn't manage to get it right ... would this be a reasonable feature to add to resilient.js?

**Comment by h2non** [comment]
Yes, it can be achieved easily. Send method cam support vadiadic arguments and infer them based of their type.

**Comment by h2non** [comment]
@ismriv are you able to provide a PR? Otherwise I can support it in a near future.

**Comment by ismriv** [comment]
Sure, I'll try to provide a PR by the end of the week.
